README.md

```diff
@@ -385,11 +386,12 @@ It's required to set any of (at least one): `Common Name` or `SAN DNS`.
 In order to prevent an issuance of a new certificate if current certificate exists in Vault's storage, we added a capability
 to return that certificate instead. To issue this feature you must set:
 
-- `min_cert_time_left`: Golang's duration format string (e.g. 24h, 23h5m20s, 10000s, etc.)
-- `store_by="hash"`
-- `store_pkey=true`
+- `min_cert_time_left` (_optional_): Golang's duration format string (e.g. 24h, 23h5m20s, 10000s, etc.). Default is 30 days.
+- `store_by="hash"` (_required_)
+- `store_pkey=true` (_required_)
+- `no_cache=false` (_required_)
 
-If certificate was successfully loaded from Vault storage, you will encounter `Loading certificate from storage` message
+`If certificate was successfully loaded from Vault storage, you will encounter `Loading certificate from storage` message
```

Comment:
is this ` not breaking anything?

Reply:
Yes it was. Thank you for noticing. Already removed on the next commit.